### PR TITLE
Review

### PR DESCRIPTION
**Pull request by msharafjr**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,396 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="description" content="Official FC Barcelona website. All news about the team, ticket sales, member services, supporters club services and information about Barça and the Club">
```
Comment (shadyvb) [inline index.html:6]: Indent

File: index.html [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+		<title>FCBarcelona | jQuery With Bootstrap 3</title>
+		<link rel="stylesheet" type="text/css" href="css/bootstrap.css">
+		<link rel="stylesheet" type="text/css" href="css/font-awesome.min.css">
+		<link rel="stylesheet" type="text/css" href="css/main.css">
+		<script src="js/html5shiv.min.js"></script>
+		<script src="js/respond.min.js"></script>
+	</head>
+	<body>
+            <!--    Navigation Bar      -->
+        <nav class="navbar navbar-default">
```
Comment (shadyvb) [inline index.html:18]: Indent

File: index.html [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+                </ul>
+            </div>
+        </div>
+            <!--    End Header logo     -->
+            <!--    schedule       -->
+        <section class="schedule-bar">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-offset-2 col-md-8">
+                        <h2>club schedule</h2>
```
Comment (shadyvb) [inline index.html]: club > Club

File: index.html [hunk 4]
```diff
@@ -0,0 +1,396 @@
...
+        </section>
+            <!--    End schedule       -->
+            <!--    Start Main Carousel      -->
+        <div id="carousel-example-generic" class="carousel slide main-carousel" data-ride="carousel">
+          <!-- Wrapper for slides -->
+          <div class="carousel-inner" role="listbox">
+            <div class="item active">
+              <img src="images/news/29252139.jpg" alt="messi">
+              <div class="carousel-caption main-carousel-caption hidden-xs">
+                <h1>FC Barcelona Get Straight Sack To Training</h1>
```
Comment (shadyvb) [inline index.html]: There shouldn't be too many H1 elements in a single page, typically a page should only have a single H1, a few H2's, and a bunch of H3's.

In H_x_, think of the _x_ as the number of permitted occurrences of the heading type.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,396 @@
...
+            <li data-target="#carousel-example-generic" data-slide-to="1"><img src="images/news/29508058.jpg" alt="enrique"></li>
+            <li data-target="#carousel-example-generic" data-slide-to="2"><img src="images/news/33518581.jpg" alt="suarez"></li>
+            <li data-target="#carousel-example-generic" data-slide-to="3"><img src="images/news/33816026.jpg" alt="doro"></li>
+          </ol>
+        </div>
+            <!--    End Main Carousel        -->
+            <!--    Start Section about -->
+        <section class="about text-center">
+            <div class="container">
+                <h2>CAMP NOU EXPERIENCE</h2>
```
Comment (shadyvb) [inline index.html]: Capitalize via styling, HTML should have normal-cased letters.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,396 @@
...
+                <p class="lead">History happens here. Live it with the Tour & Museum.</p>
```
Comment (shadyvb) [inline index.html]: `&` should be written as `&amp;` in HTML, that's why GitHub highlights it in Red.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,396 @@
...
+                </div>
+            <!--    End Special Features Carousel        -->
+            </div>
+        </section>
+            <!--    End Special Features Section    -->
+            <!--    Camp Nou Section    -->
+        <section class="new-camp-nou">
+            <h2 class="text-center">New Camp Nou</h2>
+            <div class="campnou-body">
+                <div class="container-fluid img-responsive">                        <img src="images/campnou/19910958.jpg" alt="camp nou" class="img-responsive">
```
Comment (shadyvb) [inline index.html]: `img` tag should be on a new line.

File: js/app.js [hunk 8]
```diff
@@ -0,0 +1,135 @@
+/*global $*/
+$(function () {
+    'use strict';
+
+    $(".store-kit").on('click', '.get', function () {
```
Comment (shadyvb) [inline js/app.js]: Use single quotes unless you need otherwise.

File: js/app.js [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+
+        var storKit = $(this).closest(".store-kit"),
+            amount = storKit.data('price'),
+            price = $('<p class="price center-block">' + amount + '</p>'),
```
Comment (shadyvb) [inline js/app.js]: If you're creating new elements, construct them in jQuery, not by writing HTML directly. This protects you from XSS injection problems ( a hacking method ).

Example:
`$( '<p/>', { class: 'price center-block' } )` and `$( '<a/>', { href: 'xxx', class: 'xxx' } ).text( 'Buy now' )`

**Review by shadyvb — CHANGES_REQUESTED** [review]
Fix indentation, letter cases, in all files, and take care of the other comments.

File: js/app.js (hunk 9 above)
Comment (msharafjr) [reply js/app.js]: Done

File: js/app.js (hunk 8 above)
Comment (msharafjr) [reply js/app.js]: 'Done'

File: index.html (hunk 7 above)
Comment (msharafjr) [reply index.html]: eh eli wadda el tag da henak?! :smile: 
Done

File: index.html (hunk 6 above)
Comment (msharafjr) [reply index.html]: Done

File: index.html (hunk 5 above)
Comment (msharafjr) [reply index.html]: Done

File: index.html (hunk 4 above)
Comment (msharafjr) [reply index.html]: Done

File: index.html (hunk 3 above)
Comment (msharafjr) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (msharafjr) [reply index.html:18]: Done

File: index.html (hunk 1 above)
Comment (msharafjr) [reply index.html:6]: Done

**Pull request merged**